portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,7 +84,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     String userName = request.getParameter("user");
     String comment = request.getParameter("comment-body");
     // it is allowable for the image url to be null
-    String blobKey = getBlobKey(request, "image").getKeyString();
+    BlobKey currentKey = getBlobKey(request, "image");
+    String blobKey;
```

Comment:
Maybe name this `keyString`? It's a little confusing to have a variable `blobKey` right next to a different variable of type `BlobKey`.